pipelines/pulpcore/02-install.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: install pulpcore
+  hosts:
+   - "{{ forklift_server_name }}"
+  become: True
+  vars_files:
+    - ../vars/install_base.yml
+    - ../vars/forklift_{{ pipeline_type }}.yml
+  roles:
+    - epel_repositories
+    - pulp_all_services
```

Comment:
I think this role be missing.

Reply:
no, it just needs the roles from pulp_installer and I didn't document that part yet

Reply:
now it installs it automatically